src/main/java/cz/cvut/fel/aic/simod/ridesharing/RidesharingDispatcher.java

```diff
@@ -315,12 +315,12 @@ public void saveSolution(Map<RideSharingOnDemandVehicle, DriverPlan> plans, List
 
 		try {
 			DarpSolutionPlan[] darpPlans = new DarpSolutionPlan[plans.size()];
-			int totalCost = 0;
+			double totalCost = 0;
 			for (int i = 0; i < plans.size(); i++) {
 				Entry<RideSharingOnDemandVehicle, DriverPlan> entry = (Entry<RideSharingOnDemandVehicle, DriverPlan>) plans.entrySet().toArray()[i];
 				RideSharingOnDemandVehicle vehicle = entry.getKey();
 				DriverPlan plan = entry.getValue();
-				int cost = (int) (plan.cost);
+				double cost = plan.cost;
```

Comment:
I think these costs can be of type long, right?

Reply:
DriverPlan.cost  is double and we are adding these costs so I don't see why use long instead.

Reply:
ok